tests/testthat/test-accuracy-published.R

```diff
@@ -0,0 +1,245 @@
+
+
+test_that(".OneParExpNormalPrior works as expected", {
+
+
+  mcmc_options <- crmPack::McmcOptions(
```

Comment:
please don't use `crmPack::` in any tests, because it is not necessary

Reply:
removed all "crmPack::"